.github/workflows/publish.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
     name: CD
     uses: grafana/plugin-ci-workflows/.github/workflows/cd.yml@main
     with:
+      go-version: '1.23.5'
```

Comment:
We should be able to switch to 1.23 once this PR is merged: https://github.com/actions/go-versions/pull/110